plugin/src/main/java/org/opensearch/ml/sdkclient/SdkClientFactory.java

```diff
@@ -92,16 +88,20 @@ public static SdkClient createSdkClient(Client client, NamedXContentRegistry xCo
                     throw new OpenSearchException("AWS Opensearch Service client requires a metadata endpoint and region.");
                 }
                 log.info("Using remote AWS Opensearch Service cluster as metadata store");
-                return new SdkClient(new RemoteClusterIndicesClient(createAwsOpenSearchServiceClient(remoteMetadataEndpoint, region)));
+                return new SdkClient(
+                    new RemoteClusterIndicesClient(createAwsOpenSearchServiceClient(remoteMetadataEndpoint, region, "es"))
+                );
             case SdkClientSettings.AWS_DYNAMO_DB:
                 if (Strings.isBlank(remoteMetadataEndpoint) || Strings.isBlank(region)) {
-                    throw new OpenSearchException("AWS Opensearch Service client requires a metadata endpoint and region.");
+                    throw new OpenSearchException(
+                        "AWS DynamoDB and Opensearch Serverless Service client requires a metadata endpoint and region."
+                    );
                 }
                 log.info("Using dynamo DB as metadata store");
                 return new SdkClient(
                     new DDBOpenSearchClient(
                         createDynamoDbClient(region),
-                        new RemoteClusterIndicesClient(createOpenSearchClient(remoteMetadataEndpoint))
+                        new RemoteClusterIndicesClient(createAwsOpenSearchServiceClient(remoteMetadataEndpoint, region, "aoss"))
```

Comment:
`aoss` --> Shouldn't we get that from some configuration over than hard coded?

Reply:
Heh, I just replied to the other comment with this suggestion. Will do that and update PR.